scripts/image_upload_glance.py

```diff
@@ -27,16 +27,17 @@ def parse_cmd_line():
     parser = argparse.ArgumentParser(description=__doc__,
                                      formatter_class=argparse.RawDescriptionHelpFormatter)
 
-    parser.add_argument('--image', help='Image to be uploaded to Glance', required=True)
+    parser.add_argument('--image', help='Image to be uploaded to Glance', required=False)
     parser.add_argument('--image_name_in_glance', help='Image name in Glance', required=True)
     parser.add_argument('--provider', help='Glance provider key in cfme_data', required=True)
     parser.add_argument('--disk_format', help='Disk format of image', default='qcow2')
+    parser.add_argument('--url', help='Location of image to be uploaded to Glance')
```

Comment:
It may be be better to use URL rather than Location in the help, as one could thing of a file path as giving  location too.   With the context they should get that, but I think URL is more specific.

Reply:
@jamesooden , done